src/main/java/com/openmrs/migrator/core/utilities/ConsoleUtils.java

```diff
@@ -0,0 +1,117 @@
+package com.openmrs.migrator.core.utilities;
+
+import java.io.Console;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class ConsoleUtils {
```

Comment:
Have you looked into writing unit tests for these?

Reply:
`java.io.Consol` e returns  null when is not used in the console, IDE for example. May be  if we use other `Inputstream`, but the writers of java highly  recommend  the use of  Console as  new  way  to interact with user in command line

Reply:
cool that's fine :+1: , it is not critical since it is an interface, we'll know if there are any issues